sanssouci/post_hoc_bounds.py

```diff
@@ -10,7 +10,7 @@ def max_fp(p_values, thr):
     """
     Upper bound for the number of false discoveries in a selection
 
-    * Inputs:
+    * Parameters:
```

Comment:
```suggestion
    Parameters
    -----------
```

Reply:
Sure, this will be addressed later via #5